src/Symfony/Component/Workflow/Tests/AlphabeticalEnum.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Symfony\Component\Workflow\Tests;
+
+enum AlphabeticalEnum: string
```

Comment:
Please move fixtures into a dedicated `Fixtues` sub-namespace.

Reply:
Moving to fixtures :+1:

You can already do something like this :slightly_smiling_face: 
```php
new Definition(Suit::cases(), …);
```

Reply:
> You can already do something like this

Right, but this is not what your tests are doing at the moment. 🙃